platform/docs/versioned_docs/version-3.0/platform/services/data/ToolbarService.md

```diff
@@ -210,6 +210,16 @@ to create `MeasurementTools` nested button
           toolName: 'EllipticalRoi',
         }
       },
+      // Planar Freehand tool
+      {
+        id: 'PlanarFreehandROI',
+        icon: 'tool-freehand',
+        label: 'Planar Freehand',
+        type: 'tool',
+        commandOptions: {
+          toolName: 'PlanarFreehandROI',
+        }
+      },
```

Comment:
why are you chaning versioned docs? these are archived, you need to change the root docs

Reply:
:D ok thanks